PORTAL/jobs/_utils.py

```diff
@@ -703,7 +800,7 @@ def raw(self):
 ##################################
 # OS utils
 
-def resolve_user(cfg, user=None):
+def resolve_user(cfg: Any, user: Optional[str] = None) -> str:
```

Comment:
I'm going to try to stop suggesting more specific types than `Any`.  Lingering cases can be easily found later and addressed then.

Reply:
Yeah, I think for the most part that's true.  If they lead to any refactoring bugs in the future, we can definitely handle then.

Reply:
In this specific case, this variable isn't used, so `Any` is as good as anything.